packages/ui/src/ButtonVariants.tsx

```diff
@@ -76,26 +105,35 @@ export const ButtonBase = styled(ButtonComponent)`
     pointer-events: none;
```

Comment:
We could maybe add here some flag if(!icon) {addOpacity}.
something like this could give us ability to have button with icon, and also (justIcon without hover effects), at-least i hope :)

Reply:
We can use disabled prop but than button will have pointer-events:none and opacity 0.5 regarding design.
Maybe to add separated opacity prop so we can pass disabled (to disable hover effects) and opacity=1. Then we will have just icon without hover effects?

Reply:
let's add prop iconOnly that will handle all this.

Reply:
do you have example in storybook for this button:
![image](https://user-images.githubusercontent.com/22115870/175546249-1e5a8835-4d3b-4d72-aec5-caca90566c0c.png)
we need that as well.